src/runtime/graph/graph_runtime.cc

```diff
@@ -78,18 +79,21 @@ void GraphRuntime::Init(const std::string& graph_json,
   ctxs_ = ctxs;
   this->SetupStorage();
   this->SetupOpExecs();
+  for (size_t i = 0; i < input_nodes_.size(); i++) {
+    uint32_t nid = input_nodes_[i];
+    std::string& name = nodes_[nid].name;
```

Comment:
`const` or just inline this into next line.

Reply:
Fixed.